pkg/controller/oneagent/oneagent_controller.go

```diff
@@ -345,7 +345,7 @@ func newPodSpecForCR(instance *dynatracev1alpha1.OneAgent) corev1.PodSpec {
 				Handler: corev1.Handler{
 					Exec: &corev1.ExecAction{
 						Command: []string{
-							"grep", "oneagentwatchdo", "/proc/*/stat",
+							"/bin/sh", "-c", "grep", "oneagent", "/proc/*/stat",
```

Comment:
Why did we leave out the `oneagentwatchdog` and using plain `oneagent`?

Reply:
My bad, copied the command I was using to test.